worlds/dkc3/Rom.py

```diff
@@ -0,0 +1,547 @@
+        rom.write_byte(0x3B96A9, 0x05)
+        rom.write_byte(0x3B96A8, 0xA0)
+    elif world.kong_palette_swap[player] == "chocolate":
+        rom.write_byte(0x3B96A9, 0x1D)
+        rom.write_byte(0x3B96A8, 0xA0)
+    elif world.kong_palette_swap[player] == "shadow":
+        rom.write_byte(0x3B96A9, 0x45)
+        rom.write_byte(0x3B96A8, 0xA0)
+    elif world.kong_palette_swap[player] == "red_gold":
+        rom.write_byte(0x3B96A9, 0x5D)
+        rom.write_byte(0x3B96A8, 0xA0)
+    elif world.kong_palette_swap[player] == "gbc":
+        rom.write_byte(0x3B96A9, 0x20)
+        rom.write_byte(0x3B96A8, 0x3C)
+    elif world.kong_palette_swap[player] == "halloween":
+        rom.write_byte(0x3B96A9, 0x70)
+        rom.write_byte(0x3B96A8, 0x3C)
+
+    if world.music_shuffle[player]:
+        for address in music_rom_data:
+            rand_song = local_random.choice(level_music_ids)
+            rom.write_byte(address, rand_song)
+
+    # Starting Lives
+    rom.write_byte(0x9130, world.starting_life_count[player].value)
+    rom.write_byte(0x913B, world.starting_life_count[player].value)
+
+
+    # Handle Level Shuffle Here
+    if world.level_shuffle[player]:
+        for i in range(len(active_level_list)):
+            rom.write_byte(level_dict[level_list[i]].nameIDAddress, level_dict[active_level_list[i]].nameID)
+            rom.write_byte(level_dict[level_list[i]].levelIDAddress, level_dict[active_level_list[i]].levelID)
+
+        # First levels of each world
+        rom.write_byte(0x34BC3E, (0x32 + level_dict[active_level_list[0]].levelID))
+        rom.write_byte(0x34BC47, (0x32 + level_dict[active_level_list[5]].levelID))
+        rom.write_byte(0x34BC4A, (0x32 + level_dict[active_level_list[10]].levelID))
+        rom.write_byte(0x34BC53, (0x32 + level_dict[active_level_list[15]].levelID))
+        rom.write_byte(0x34BC59, (0x32 + level_dict[active_level_list[20]].levelID))
+        rom.write_byte(0x34BC5C, (0x32 + level_dict[active_level_list[25]].levelID))
+        rom.write_byte(0x34BC65, (0x32 + level_dict[active_level_list[30]].levelID))
+        rom.write_byte(0x34BC6E, (0x32 + level_dict[active_level_list[35]].levelID))
+
+        # Cotton-Top Cove Boss Unlock
+        rom.write_byte(0x34C02A, (0x32 + level_dict[active_level_list[14]].levelID))
+
+        # Kong-Fused Cliffs Unlock
+        rom.write_byte(0x34C213, (0x32 + level_dict[active_level_list[25]].levelID))
+        rom.write_byte(0x34C21B, (0x32 + level_dict[active_level_list[26]].levelID))
+
+    if world.goal[player] == "knautilus":
+        # Swap Kastle KAOS and Knautilus
+        rom.write_byte(0x34D4E1, 0xC2)
+        rom.write_byte(0x34D4E2, 0x24)
+        rom.write_byte(0x34D551, 0xBA)
+        rom.write_byte(0x34D552, 0x23)
+
+        rom.write_byte(0x32F339, 0x55)
+
+
+    from Main import __version__
+    rom.name = bytearray(f'BM{__version__.replace(".", "")[0:3]}_{player}_{world.seed:11}\0', 'utf8')[:21]
```

Comment:
I think 'BM' refers to Berserker's Multiworld and is used to indicate that the game is LTTP.